.npmignore

```diff
@@ -0,0 +1,3 @@
+.*
+test
+coverage
```

Comment:
Not sure why you are using `.npmignore` and not `.gitignore`. Would it be possible to use only `.gitignore`?

Reply:
`.npmignore` is for when it is time for you to `npm publish` whereas `.gitignore` is for any time making a commit. One can have files that don't go in the repo but do go into npm and vice versa (e.g., we don't need `test` in the npm package).

If you prefer, one can use [`files`](https://docs.npmjs.com/files/package.json#files). (One disadvantage of this I seem to recall though is that it causes problems when trying to point `package.json` to Github, e.g., a fork while one is waiting on an official npm release; IIRC, npm might not let you pick up distribution files, as I recall needing to add `dist` to this field.)

Reply:
Ok, but let's just put `coverage` into `.gitignore` and keep `test` cause
- I'm a fan of having the same content on npm and git (can be confusing sometimes)
- Maybe never relevant for this package, but in rare case the tests are also useful (e.g. [this package](https://github.com/rdfjs-base/data-model/blob/master/test/test.js) allows running the spec tests against your own implementation)

Reply:
But if we don't use one of the approaches, the coverage directories (as well as `test`) will get passed on with npm releases...

Reply:
Ok, sure. Sorry, posted before seeing your comment.

Reply:
I've amended `.npmignore` to avoid ignoring `test` (`coverage` was actually already in `.gitignore` so nothing to do there).